cicd/deploy.py

```diff
@@ -0,0 +1,90 @@
+import requests
+import json
+import os
+# Замените эти переменные на ваши значения
+PORTAINER_TOKEN = os.getenv("PORTAINER_TOKEN")
+NEXUS_URL = os.getenv("NEXUS_URL")
+CLICKHOUSE_URL = os.getenv("CLICKHOUSE_URL")
+CLICKHOUSE_USER = os.getenv("CLICKHOUSE_USER")
+CLICKHOUSE_PASSWORD = os.getenv("CLICKHOUSE_PASSWORD")
+PORTAINER_HOST = os.getenv("PORTAINER_HOST")
+PORTAINER_PORT = os.getenv("PORTAINER_PORT")
+STRATUM_BRANCH = os.getenv("STRATUM_BRANCH")
+REDROCK_API_URL = os.getenv("REDROCK_API_URL")
+REDROCK_API_KEY = os.getenv("REDROCK_API_KEY")
+NODE_CHAIN = os.getenv("NODE_CHAIN")
+NODE_HOST = os.getenv("NODE_HOST")
+
+if NODE_CHAIN == 'PoolTest':
+    STACK_ID = os.getenv("POOLTEST_STRATUM_STACK_ID")
+    ENDPOINT_ID = os.getenv("POOLTEST_STRATUM_ENDPOINT_ID")
+    NODE_PORT = os.getenv("POOLTEST_NODE_PORT")
+    NODE_KEY = os.getenv("POOLTEST_NODE_KEY")
+    COINBASE_TYPE = os.getenv("POOLTEST_COINBASE_TYPE")
+    COINBASE_VALUE = os.getenv("POOLTEST_COINBASE_VALUE")
+    POOL_PORT = os.getenv("POOLTEST_POOL_PORT")
+    TRANSLATOR_PORT = os.getenv("POOLTEST_TRANSLATOR_PORT")
+    METRICS_PORT = os.getenv("POOLTEST_METRICS_PORT")
+    
+elif NODE_CHAIN == 'TestNet':
+    STACK_ID = os.getenv("TESTNET_STRATUM_STACK_ID")
+    ENDPOINT_ID = os.getenv("TESTNET_STRATUM_ENDPOINT_ID")
+    NODE_PORT = os.getenv("TESTNET_NODE_PORT")
+    NODE_KEY = os.getenv("TESTNET_NODE_KEY")
+    COINBASE_TYPE = os.getenv("TESTNET_COINBASE_TYPE")
+    COINBASE_VALUE = os.getenv("TESTNET_COINBASE_VALUE")
+    POOL_PORT = os.getenv("TESTNET_POOL_PORT")
+    TRANSLATOR_PORT = os.getenv("TESTNET_TRANSLATOR_PORT")
+    METRICS_PORT = os.getenv("TESTNET_METRICS_PORT")
+
+elif NODE_CHAIN == 'MainNet':
+    STACK_ID = os.getenv("MAINNET_STRATUM_STACK_ID")
+    ENDPOINT_ID = os.getenv("MAINNET_STRATUM_ENDPOINT_ID")
+    NODE_PORT = os.getenv("MAINNET_NODE_PORT")
+    NODE_KEY = os.getenv("MAINNET_NODE_KEY")
+    COINBASE_TYPE = os.getenv("MAINNET_COINBASE_TYPE")
+    COINBASE_VALUE = os.getenv("MAINNET_COINBASE_VALUE")
+    POOL_PORT = os.getenv("MAINNET_POOL_PORT")
+    TRANSLATOR_PORT = os.getenv("MAINNET_TRANSLATOR_PORT")
+    METRICS_PORT = os.getenv("MAINNET_METRICS_PORT")
+
+else:
+    print("Variable NODE_CHAIN is not valid")
```

Comment:
Log or handle the invalid NODE_CHAIN case before exiting to provide more context.